en-US/Docs/Loopback.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: docs
+title: Communicating with localhost (loopback)
+description: How to turn on Embedded Mode for Windows desktop and Windows Mobile editions
+keyword: windows runtime, windows iot, localhost, loopback
+permalink: /en-US/Docs/Loopback.htm
+lang: en-US
+---
+
+# Communicating with localhost (loopback)
+
+On **Windows IoT Core** If you want to create a TCP/IP connection between 2 processes running on the same device and one of them is a UWP app you must enable localhost loopback.
+
+## Loopback and the debugger 
+By default, running under the debugger enables outbound loopback automatically for that debug session only.  You shouldn’t have to do anything as long as the loopback checkbox is checked in the debugger settings for your startup project.  If you want to implement a socket listener the you must enable localhost loopback for inbound connections (see below).
+ 
+## Enabling the loopback policy
+The localhost loopback policy for **Windows IoT Core** is controlled by the following registry key:
+
+        [HKEY_LOCAL_MACHINE\system\currentcontrolset\services\mpssvc\parameters]
+            "IoTInboundLoopbackPolicy"=dword:00000001
+
+This registry key must be set and if you change it you must reboot.  The localhost loopback policy should be enabled by default on **Windows IoT Core**
+
+To verify that the value is set execute the following command on the **Windows IoT Core** device:
+
+        reg query hklm\system\currentcontrolset\services\mpssvc\parameters /v IoTInboundLoopbackPolicy
+
+To enable the policy execute the following command on the **Windows IoT Core** device:
+
+        reg add hklm\system\currentcontrolset\services\mpssvc\parameters /v IoTInboundLoopbackPolicy /t REG_DWORD /d 1
+ 
+
+## Enabling loopback for a UWP application
+Before you can enable loopback for an application you will need the package family name.  You can find the package family name for an installed application by running **iotstartup list**.  If the **iotstartup list** entry for the application is IoTCoreDefaultApp\_1w720vyc4ccym!App then the package family name is IoTCoreDefaultApp\_1w720vyc4ccym
+
+To enable loopback for client connections use **CheckNetIsolation.exe -a**.  This will enable the application to make outbound connections to a server.

```

Comment:
To enable ... to make output connections to a server until the device restarts or CheckNetIsolation is no longer running (see below for details on making sure this runs on startup)